lib/Tunnel.js

```diff
@@ -1,158 +1,148 @@
-var url = require('url');
-var EventEmitter = require('events').EventEmitter;
-var axios = require('axios');
-var debug = require('debug')('localtunnel:client');
-
-var TunnelCluster = require('./TunnelCluster');
-
-var Tunnel = function(opt) {
-    if (!(this instanceof Tunnel)) {
-        return new Tunnel(opt);
-    }
-
-    var self = this;
-    self._closed = false;
-    self._opt = opt || {};
-
-    self._opt.host = self._opt.host || 'https://localtunnel.me';
-};
-
-Tunnel.prototype.__proto__ = EventEmitter.prototype;
-
-// initialize connection
-// callback with connection info
-Tunnel.prototype._init = function(cb) {
-    var self = this;
-    var opt = self._opt;
-
-    var params = {
-        responseType: 'json'
+/* eslint-disable consistent-return, no-underscore-dangle */
+
+const url = require('url');
+const { EventEmitter } = require('events');
+const axios = require('axios');
+const debug = require('debug')('localtunnel:client');
+
+const TunnelCluster = require('./TunnelCluster');
+
+module.exports = class Tunnel extends EventEmitter {
+  constructor(opts = {}) {
+    super(opts);
+    this.opts = opts;
+    this.closed = false;
+    if (!this.opts.host) this.opts.host = 'https://localtunnel.me';
```

Comment:
Please use `{ }` for all `if` statements.

Reply:
Done.